tests/conditional_compilation/test_cc.py

```diff
@@ -78,38 +81,51 @@ def test_minimized_pkg(test_id, model, openvino_root_dir, artifacts):  # pylint:
 
 
 @pytest.mark.dependency(depends=["cc_collect", "minimized_pkg"])
-def test_infer(test_id, model, artifacts):
+def test_infer(test_id, models, artifacts):
     """Test inference with conditional compiled binaries."""
     out = artifacts / test_id
     minimized_pkg = out / "install_pkg"
-    return_code, output = run_infer(model, f"{out}_cc.npz", minimized_pkg)
+    infer_out_dir_cc = artifacts / f"{test_id}_cc" / "inference_result"
+    infer_out_dir_cc.mkdir(parents=True, exist_ok=True)
```

Comment:
This operation really needed? Isn't create necessary directory by `run_infer`?

Reply:
@akoryach please, move folder creation in `run_infer` function (not `infer_tool.py`)